tests/bits/find_cell_5.output

```diff
@@ -1,3 +1,3 @@
 
-DEAL::3.462
-DEAL::<0.750000 0.625000 0.625000> <0.875000 0.625000 0.625000> <0.750000 0.750000 0.625000> <0.875000 0.750000 0.625000> <0.750000 0.625000 0.750000> <0.875000 0.625000 0.750000> <0.750000 0.750000 0.750000> <0.875000 0.750000 0.750000> 
+DEAL::3.455
+DEAL::<0.625000 0.625000 0.625000> <0.750000 0.625000 0.625000> <0.625000 0.750000 0.625000> <0.750000 0.750000 0.625000> <0.625000 0.625000 0.750000> <0.750000 0.625000 0.750000> <0.625000 0.750000 0.750000> <0.750000 0.750000 0.750000> 
```

Comment:
why does this test (and the others below) change output? Is the point in question right on the interface between cells?